### ISSUE TITLE
Dashboard live reload

### ISSUE THREAD

**Issue by lyda** [daveshanley/vacuum#409]
Would there be any interest in a live reload feature for the dashboard command?  The [fsnotify](https://github.com/fsnotify/fsnotify) library seems like it would work.

**Comment by daveshanley** [comment]
Yeah, absolutely! it's something I have been meaning to add, however please feel free to submit a PR!

**Comment by lyda** [comment]
Awesome!  OK, I'll see what I can do.

**Comment by daveshanley** [comment]
Fixed in `v0.18.0`

https://quobix.com/vacuum/commands/dashboard/#watch-mode
